cowprotocol/tenderly-watch-tower
# Add docker build push pipeline

# Description
Builds and pushes the docker image on push to the `main` branch and on creating a release.

# Changes

- [x] Creates a new Github Actions workflow

## How to test

1. Make sure the `Dockerfile` exists in the parent directort
2. Push to main branch/create a release

## Related Issues

Depends on #52
